examples/3d/lightmaps.rs

```diff
@@ -0,0 +1,55 @@
+//! Rendering a schene with baked lightmaps.
+
+use bevy::pbr::Lightmap;
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(AmbientLight {
+            color: Color::WHITE,
+            brightness: 0.2,
+        })
+        .add_systems(Startup, setup)
+        .add_systems(Update, add_lightmaps_to_meshes)
+        .run();
+}
+
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    commands.spawn(SceneBundle {
+        scene: asset_server.load("models/CornellBox/CornellBox.glb#Scene0"),
+        ..SceneBundle::default()
+    });
+}
```

Comment:
Typically in bevy examples the camera is spawned manually rather than included in the gltf file, allowing users to more easily modify/test things involving the camera entity.

Reply:
Personally I find it easier to modify in glTF because I can use Blender :) But sure, I can change it.

Reply:
It's easy to move the camera in blender, but it makes it hard to to modify other bevy specific camera settings (like enabling hdr, adding a camera controller, adding a prepass, etc...) as it would require the user to setup a system to modify the camera after startup.

Reply:
This is now done manually.